.github/workflows/canary.yml

```diff
@@ -0,0 +1,64 @@
+name: Canary Release
```

Comment:
I guess we need the 2nd workflow for release? this way we can release using `changeset` as well. What do you think @IvanGoncharov ? 
(Changesets collects all changeset files and keep an updated Release PR, so when it's merged, you get the actual version bump and NPM release)

Reply:
We discuss canary release so let's address that and see how it goes.